package.json

```diff
@@ -2,22 +2,24 @@
   "name": "envalid",
   "version": "6.0.2",
   "description": "Validation for your environment variables",
-  "main": "src/envalid.js",
-  "types": "src/envalid.d.ts",
-  "browser": "src/envalidWithoutDotenv.js",
-  "files": [
-    "src/"
-  ],
+  "main": "dist/index.js",
+  "module": "dist/envalid.esm.js",
+  "typings": "dist/index.d.ts",
   "engines": {
     "node": ">=8.12"
   },
   "engineStrict": true,
+  "files": [
+    "dist",
+    "src"
```

Comment:
I personally like having it distributed in dependencies for debugging– is there any downside to including it, besides using a bit of disk space?

Reply:
Just disk usage, yeah (I'm a bit wary since people seem sensitive to it (https://github.com/facebook/jest/issues/6266)), probably not an issue here tho